gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -0,0 +1,73 @@
+require 'active_record'
+require 'active_support/core_ext'
+require 'linux_admin'
+require 'pathname'
+require 'util/miq-password'
+require 'fileutils'
+require 'appliance_console/errors'
+require 'appliance_console/logging'
+require 'appliance_console/logical_disk_volume'
+require 'appliance_console/prompts'
+require 'appliance_console/utilities'
+
+RAILS_ROOT ||= Pathname.new(__dir__).join("../../../")
+
+module ApplianceConsole
+  class LogfileConfiguration
+    LOGFILE_DIRECTORY = Pathname.new("/var/www/miq/vmdb/log").freeze
+    LOGFILE_NAME = "miq_logs".freeze
+
+    attr_accessor :disk
+
+    include ApplianceConsole::Logging
+
+    def initialize
+      self.disk = nil
+    end
+
+    def activate
```

Comment:
Can we put a confirmation page that prints the disk it will use in here before doing the work?